bucket/workspacer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.9.9",
+    "description": "a tiling window manager for Windows 10",
+    "homepage": "https://www.workspacer.org/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://workspacer.blob.core.windows.net/installers/stable/workspacer-stable-0.9.9.msi"
```

Comment:
```suggestion
            "url": "https://workspacer.blob.core.windows.net/installers/stable/workspacer-stable-0.9.9.msi"
            "hash": "647b171fa8420ccd07e997a50d440cdbdcdb1cd3472fe50dda8a120ae70cd58d",
```

Reply:
Fixed.

Reply:
@sitiom Opps, there should be no , after the end of hash. Could you remove that? Sorry about that.